src/components/RubyGemsVersionPicker.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { getRubyGemsRevisions } from '../api/clearlyDefined'
+import { Typeahead } from 'react-bootstrap-typeahead'
+
+export default class RubyGemsVersionPicker extends Component {
+  static propTypes = {
+    onChange: PropTypes.func,
+    request: PropTypes.object.isRequired,
+    defaultInputValue: PropTypes.string
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = { customValues: [], options: [] }
+    this.onChange = this.onChange.bind(this)
+    this.filter = this.filter.bind(this)
+  }
+
+  componentDidMount() {
+    this.getOptions('')
+  }
+
+  async getOptions(value) {
+    try {
+      const { namespace, name } = this.props.request
+      const path = namespace ? `${namespace}/${name}` : name
```

Comment:
is there every a namespace?

Reply:
There is no namespace so I have removed any rubygem namespace references in both the service and website